app/app-controllers/error-screen/errorController.js

```diff
@@ -89,7 +89,13 @@ define([
             self.viewPlugin.on('error:loaded', function() {
                 self.show();
             });
-            self.errorType = errorType;
+
+            if (eventArgs.error.code == -1001) { //timeout
```

Comment:
Once the Astro PR is merged we should change this to use the new js "constants" :)